# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/sessionstore/test/unit/xpcshell.ini
# Commit: 2a8633af0279
# Full Hash: 2a8633af0279379685ba781c6c469e8b07ec13f4
# Author: Mathew Hodson <mathew.hodson@gmail.com>
# Date: 2022-03-26 09:28:42
# Regressor Bug: 1752853
# File Overlap Count: 1
# Description:
#   Bug 1752853 - Stop using a worker to write session store. r=Gijs,Standard8
#   
#   Remove now unused pref "browser.sessionstore.max_write_failures".
#   Rename and simplify test_shutdown_cleanup.js to make it more clear that
#   it is testing the isFinalWrite option.
# ==============================================================================

diff -r dd81d58d9d5f -r 2a8633af0279 browser/components/sessionstore/test/unit/xpcshell.ini
--- a/browser/components/sessionstore/test/unit/xpcshell.ini	Fri Mar 25 19:19:15 2022 +0000
+++ b/browser/components/sessionstore/test/unit/xpcshell.ini	Fri Mar 25 19:42:36 2022 +0000
@@ -8,9 +8,9 @@
   data/sessionstore_valid.js
 
 [test_backup_once.js]
+[test_final_write_cleanup.js]
 [test_histogram_corrupt_files.js]
 [test_migration_lz4compression.js]
-[test_shutdown_cleanup.js]
 [test_startup_nosession_async.js]
 [test_startup_session_async.js]
 [test_startup_invalid_session.js]
